src/System.Device.Gpio/System/Device/Gpio/Drivers/LibgpiodDriver.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace System.Device.Gpio.Drivers
+{
+    public class LibgpiodDriver : GpioDriver
+    {
+        private const string GpioGet = "gpioget";
+        private const string GpioChip0 = "gpiochip0 ";
+        private const string GpioSet = "gpioset";
+        public int _waitMilliSeconds { get; set;}
+        private Dictionary<int, PinMode> _pinNumperToPinModes;
+        private Dictionary<int, Process> _pinNumperToProcess;
+        protected internal override int PinCount => throw new PlatformNotSupportedException("This driver is generic so it can not enumerate how many pins are available.");
+
+        public LibgpiodDriver() {
+            _pinNumperToPinModes = new Dictionary<int, PinMode>();
+            _pinNumperToProcess = new Dictionary<int, Process>();
+            _waitMilliSeconds = (int)TimeSpan.FromSeconds(1).TotalMilliseconds;
```

Comment:
I'd initialize those directly in the fields, for the _waitMilliSeconds I'd just put 1000

Reply:
moved to the field, about setting 1000 for me it is more readeable for illustrating it is one second

Reply:
Up to you, but we use numbers everywhere else I'm aware of. You can write like `1_000` so it's easier to read than `1000`

Also if performance was an issue (generally in libraries we assume it might be) we would not do this because it's extra work at runtime.